src/docs/sql-editor/nft-tables/index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: NFT Tables overview
+author: Santiment Team
+date: 2023-01-30
+description: Information about Non-Fungible Token (NFT) tables stored in Santiment Queries.
+---
+
+## Overview
+NFT stands for Non-Fungible Token, which is a digital asset that represents ownership of a unique item or piece of content, like art, music, video, or tweets. NFTs are different from cryptocurrencies because each NFT is unique and can't be exchanged one-to-one, while cryptocurrencies are interchangeable. 
+
+NFTs are made using blockchain technology, which provides a secure and transparent record of ownership and transfer, allowing creators and owners to prove ownership, authenticity, and rarity, potentially increasing the value of their digital assets over time. 
+
+In **Santiment Queries**, we collect data for all NFTs from exchanges on: 
+
+### Ethereum:
+
+- Cryptopunks
+- Foundation
+- Looksrare
+- Opensea
+- Rarible
+- Superrare
+
+### Polygon:
+- Opensea_polygon
+
+
+This information gives you the ability to easily track an NFT of interest or an entire collection, as well as investigate past prices and owners.
+
+## List of nft tables
+
+All our tables that contain NFT-related data have the string 'nft' in their name.
+So one easy way to find them yourself is by running the following query:
+```sql
+SHOW TABLES LIKE '%nft%'
+```
+```
+┌─name───────────────────┐
+| nft_trades             |
+| nft_tokens_metadata    |
+| intraday_nft_metrics   |
+└────────────────────────┘
+```
+
+## Exploration of the tables
+
+> We can inspect the tables with the commands we learnt in [Exploration](/sql-editor/exploration/)
+
+Below we have listed all the nft tables with thier columns, a brief descriptions and some example queries which you can use to explore the tables.
+
+### nft trades
+
+The `nft_trades` is a table designed to store information about the NFT trades that we have collected. It has the following columns:
+
+- **dt** (*DateTime*): When the transaction happened.
+
+- **blockchain** (*String*): On which blockchain.
+
+- **tx_hash** (*String*): Unique identifier that is generated when the transaction was executed.
+
+- **log_index** (*UInt32*): Log index of trade event. 
+
+- **platform** (*String*): Name of platform where trades occured such as opensea, cryptopunks and etc. 
+
+- **seller_address** (*String*): Address who sold the nft.
+
+- **buyer_address** (*String*): Address who bought the nft.
+
+- **nft_contract_address** (*String*): Address of the nft's collection.
+
+- **currency_contract_address** (*Nullable(String)*): Address of ERC20 token or empty if native ether is used.
```

Comment:
Identifier of the currency used in the trade. It is empty if native coin was used (ether or matic), otherwise it is the address of the smart contract of the ERC20 token.